src/ast/rewriter/seq_rewriter.cpp

```diff
@@ -2788,6 +2995,19 @@ br_status seq_rewriter::mk_str_to_regexp(expr* a, expr_ref& result) {
     return BR_FAILED;
 }
 
+/*
```

Comment:
yes, such comments on the functionality are really nice and missing in too many instances.